docs/types/value-types.rst

```diff
@@ -584,6 +584,9 @@ from integer checks at runtime that the value lies inside the range of the enum
 Enums require at least one member, and its default value when declared is the first member.
 Enums cannot have more than 256 members.
 
+Using ``type(NameOfEnum).min`` and ``type(NameOfEnum).max`` yields the smallest
+(always ``0`` at this point) and respectively largest value of the given enum.
```

Comment:
Maybe move this after the next paragraph (then you also do not explain what `min` is)?

```suggestion
Using ``type(NameOfEnum).min`` and ``type(NameOfEnum).max`` you can get the smallest
and respectively largest value of the given enum.
```